src/base/bittorrent/sessionimpl.cpp

```diff
@@ -5741,6 +5747,12 @@ void SessionImpl::handleExternalIPAlert(const lt::external_ip_alert *p)
     }
 }
 
+void SessionImpl::handleSessionErrorAlert(const lt::session_error_alert *p) const
+{
+    LogMsg(tr("Session encountered a serious error. Reason: \"%1\"")
```

Comment:
Maybe it's worth clarifying what kind of session we are talking about?..

Reply:
I thought about `Libtorrent session encountered a serious error` but I don't think it helps... anyone got better ideas?

Reply:
BitTorrent session?

Reply:
OK, done.